docs/source/users/checkpointing.rst

```diff
@@ -37,6 +37,34 @@ There is also a seperate tool for working with checkpoint files
 
     fitbenchmarking-cp report --help
 
+.. list-table:: options table for checkpointing
+   :widths: 20, 10, 30, 40
+   :header-rows: 1
+
+   * - Option
+     - Flags
+     - Actions
+     - Help
+   * - *Debug mode*
+     - ``-d``
+     - ``--debug-mode``
+     - | Enable debug mode (prints traceback).
+   * - *File name*
+     - ``-f``
+     - ``--debug-mode``
+     - | The path to a fitbenchmarking checkpoint file.
+       | If omitted, this will be taken from the options
+       | file.
+   * - *Options file*
+     - ``-o``
+     - ``--options-file``
+     - | The path to a fitbenchmarking options file
+   * - *Strategy*
+     - ``-s``
+     - ``--strategy``
+     - | The merge strategy to use when dealing with 
+       | conflicts. Selecting accuracy and runtime will
+       | select for the lowest conflicting runs
```

Comment:
This was a request from Andrew. Originally the flags were in a table created in word and the image of that table was displayed in the docs. Anytime a new flag was added, someone had to recreate the table and add the new row and update the image. Doing it this way means that it is easier for developers to add flags later on. I was creating the table for the fitbenchmarking options. I added the `-rt` flag presuming that PR 1165 will be merged before this PR. So, I put the `-s` strategy flag in that table and realised that there would be two `-s` flags. So, It is better to separate the functionality into two tables. I added the table so that it is easier to add rows to the table when new functionality is added to checkpointing.